pkg/jobrunaggregator/releasebigqueryloader/table_creator.go

```diff
@@ -29,7 +29,7 @@ func (r *allReleaseTableCreatorOptions) Run(ctx context.Context) error {
 			return err
 		}
 	} else {
-		fmt.Fprintf(os.Stderr, "table already exists: %s\n", jobrunaggregatorlib.ReleaseTableName)
+		fmt.Fprintf(os.Stdout, "table already exists: %s\n", jobrunaggregatorlib.ReleaseTableName)
```

Comment:
I'm less sure about my advice now, I thought this was like an idempotent thing that was run every time we used the main commands. If it's a separate sub-command, stderr might make some sense. In any case if nobody objects stdout is still probably good. Apologies for my confusion.

Reply:
no worries.  I figure since these commands are run once to build up tables, stdout is fine.